packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -1490,6 +1492,8 @@ export const Dropdown: ComponentWithAs<'div', DropdownProps> &
     return () => {
       clearStartingString.cancel();
       clearA11ySelectionMessage.cancel();
+      context.target?.defaultView.clearTimeout(timeoutFocusTriggerButton.current);
+      context.target?.defaultView.clearTimeout(timeoutScrollPosition.current);
     };
   }, [clearA11ySelectionMessage, clearStartingString]);
```

Comment:
should I put into dependencyArray the "context" ?

Reply:
in this case probably extract the defaultView, and use that as the depedency instead, since you're just using the `document` to clear a timeout.

Also the refs should be dependnecies too for correctness